# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libtirpc/uintptr_t.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtirpc-1.0.2/src/xdr_sizeof.c.vanilla	2017-12-25 16:59:41.736454003 +0000
+++ libtirpc-1.0.2/src/xdr_sizeof.c	2017-12-25 17:00:36.440452001 +0000
@@ -39,6 +39,7 @@
 #include <rpc/xdr.h>
 #include <sys/types.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include "un-namespace.h"
 
 /* ARGSUSED */
